docs/architecture-decisions/0001-record-architecture-decisions.md

```diff
@@ -16,4 +16,4 @@ We will use Architecture Decision Records, as described by Michael Nygard in thi
 
 ## Consequences
 
-See Michael Nygard's article, linked above. For a lightweight ADR toolset, see Nat Pryce's _adr-tools_ at [https://github.com/npryce/adr-tools](https://github.com/npryce/adr-tools).
\ No newline at end of file
+See Michael Nygard's article, linked above. For a lightweight ADR toolset, see Nat Pryce's _adr-tools_ at [https://github.com/npryce/adr-tools](https://github.com/npryce/adr-tools).
```

Comment:
Non blocking question: why don't I see the white space?

Reply:
I think because by default GitHub expects to see a new line at the end of a file.

Not having a new line at the end is unexpected and results in an icon like this in the diff:
<img width="53" alt="Screenshot 2019-04-04 at 14 10 37" src="https://user-images.githubusercontent.com/2804149/55558243-90e4f600-56e3-11e9-94f5-dedbb880f6bb.png">